addon/styles/pix-design-tokens/_colors.scss

```
@@ -1,140 +1,180 @@
-$pix-secondary-certif: #20B4AF;
-$pix-secondary-certif-50: #17817E;
-$pix-secondary-certif-60: #126765;
-$pix-secondary-certif-70: #0E4D4C;
-$pix-secondary-certif-80: #093432;
-
-// Secondary Pix Orga
-$pix-secondary-orga-5: #D5F9FF;
-$pix-secondary-orga-10: #AAF4FF;
-$pix-secondary-orga-20: #80EEFF;
-$pix-secondary-orga-30: #55E8FF;
-$pix-secondary-orga: #00DDFF;
-$pix-secondary-orga-50: #00C1DF;
-$pix-secondary-orga-60: #00A6BF;
-$pix-secondary-orga-70: #008A9F;
-$pix-secondary-orga-80: #006E80;
-
-// Tertiary
-$pix-tertiary-5: #EBE1FF;
-$pix-tertiary-10: #D8C2FF;
-$pix-tertiary-20: #C4A4FF;
-$pix-tertiary-30: #B186FF;
-$pix-tertiary: #9D67FF;
-$pix-tertiary-50: #8A49FF;
-$pix-tertiary-60: #6712FF;
-$pix-tertiary-70: #4E00DB;
-$pix-tertiary-80: #3B00A4;
-
-// Success
-$pix-success-5: #ECFDF5;
-$pix-success-10: #D1FAE5;
-$pix-success-20: #A7F3D0;
-$pix-success-30: #6EE7B7;
-$pix-success: #34D399;
-$pix-success-50: #27B481;
-$pix-success-60: #14865D;
-$pix-success-70: #176C4D;
-$pix-success-80: #104834;
-
-// Warning
-$pix-warning-5: #FFF9E6;
-$pix-warning-10: #FFF1C5;
-$pix-warning-20: #FFE381;
-$pix-warning-30: #FFD235;
-$pix-warning-40: #FFBE00;
-$pix-warning-50: #EAA600;
-$pix-warning-60: #D87016;
-$pix-warning-70: #A95800;
-$pix-warning-80: #874D00;
-
-// Error
-$pix-error-5: #FEF2F2;
-$pix-error-10: #FEE2E2;
-$pix-error-20: #FECACA;
-$pix-error-30: #FCA5A5;
-$pix-error-40: #F87171;
-$pix-error-50: #EF4444;
-$pix-error-60: #DC2626;
-$pix-error-70: #B91C1C;
-$pix-error-80: #991B1B;
-$pix-error-90: #861212;
-
-// Neutral
-$pix-neutral-0: #FFFFFF;
-$pix-neutral-5: #FAFBFC;
-$pix-neutral-10: #F4F5F7;
-$pix-neutral-15: #EAECF0;
-$pix-neutral-20: #DFE1E6;
-$pix-neutral-22: #C1C7D0;
-$pix-neutral-25: #A5ADBA;
-$pix-neutral-30: #97A0AF;
-$pix-neutral-35: #8993A4;
-$pix-neutral-40: #7A869A;
-$pix-neutral-45: #6B778C;
-$pix-neutral-50: #5E6C84;
-$pix-neutral-60: #505F79;
-$pix-neutral-70: #344563;
-$pix-neutral-80: #253858;
-$pix-neutral-90: #172B4D;
-$pix-neutral-100: #091E42;
-$pix-neutral-110: #07142E;
-
-// Shades
+//// OLD - in discussion with Design Team
 $pix-shades-100: #000000;
 
-// Domain
-$pix-information-dark: #F24645;
-$pix-information-light: #F1A141;
-$pix-content-dark: #1A8C89;
-$pix-content-light: #52D987;
-$pix-communication-dark: #3D68FF;
-$pix-communication-light: #12A3FF;
-$pix-security-dark: #AC008D;
-$pix-security-light: #FF3F94;
-$pix-environment-dark: #5E2563;
-$pix-environment-light: #564DA6;
-
 //// GRADIENT
-$pix-primary-app-gradient: linear-gradient(91.59deg, #3D68FF 0%, #8845FF 100%);
-$pix-secondary-app-gradient: linear-gradient(91.59deg, #FEDC41 0%, #FF9F00 100%);
-$pix-primary-certif-gradient: linear-gradient(91.59deg, #52D987 0%, #1A8C89 100%);
-$pix-primary-orga-gradient: linear-gradient(91.59deg, #00DDFF 0%, #0095C0 100%);
-$pix-secondary-orga-gradient: linear-gradient(91.59deg, #0D7DC4 0%, #213371 100%);
+$pix-primary-app-gradient: linear-gradient(91.59deg, #3d68ff 0%, #8845ff 100%);
```

```suggestion
$pix-primary-app-gradient: linear-gradient(91.59deg, var(----pix-communication-dark) 0%, #8845ff 100%);
```

Essayer d'utiliser les variables quand c'est possible ?
Dommage, j'ai l'impression que `#8845ff` n'est pas une valeur tokenisée :/

---

> Les gradients ne sont pas encore prêts côté design, ils sont OK pour les mettre à jour dans un second temps.

De mémoire, l'intention design de la précédente version des tokens était que ces couleurs ne doivent pas être des variables 😅

---

Ah ok, je note ^^